extern.h

```diff
@@ -411,7 +412,7 @@ struct config_list {
 	TAILQ_HEAD(, config) cf_list;
 };
 
-struct config_list *config_parse(const char *, const struct environment *);
+struct config_list *config_parse(const char *, const struct environment *, struct macro_list *);
```

Comment:
`struct environment` is by convention the last argument

Reply:
I am glad I learn this convention, I always struggle to choose argument order.